static/app/views/detectors/components/details/common/ongoingIssues.tsx

```diff
@@ -1,19 +1,80 @@
+import {LinkButton} from 'sentry/components/core/button/linkButton';
+import {Flex} from 'sentry/components/core/layout';
+import EmptyStateWarning from 'sentry/components/emptyStateWarning';
 import ErrorBoundary from 'sentry/components/errorBoundary';
-import {SimpleTable} from 'sentry/components/tables/simpleTable';
+import GroupList from 'sentry/components/issues/groupList';
+import Panel from 'sentry/components/panels/panel';
+import PanelBody from 'sentry/components/panels/panelBody';
 import Section from 'sentry/components/workflowEngine/ui/section';
 import {t} from 'sentry/locale';
+import {getUtcDateString} from 'sentry/utils/dates';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePageFilters from 'sentry/utils/usePageFilters';
+
+interface Props {
+  detectorId: string;
+  query?: Record<string, any>;
+}
+
+function EmptyMessage() {
+  return (
+    <Panel>
+      <PanelBody>
+        <EmptyStateWarning small withIcon={false}>
+          {t('No issues found for this monitor')}
```

Comment:
Tiny suggestion, I wonder if this might be misleading. It's more that there are no _open_ issues in the time range, not none whatsoever